app/src/main/java/cm/aptoide/pt/home/BundlesAdapter.java

```diff
@@ -198,4 +207,14 @@ public synchronized void updateEditorials() {
       }
     }
   }
+
+  public void showLoadMoreError() {
+    bundles.add(errorBundle);
+    notifyItemInserted(bundles.size());
```

Comment:
Should be bundles.size() - 1, although it's interesting that it works.

Reply:
Also, even though this proably won't ever be a problem, it would be safer to do:

remove(bundles.indexOf(errorBundle))
add(bundles.indexOf(errorBundle))

Because this way you guarantee that you're notifying the correct item and you don't assume that it's at the end.

Remove method already exists within the class, although the add would need to be added for individual items.